thoth/python/__init__.py

```diff
@@ -17,18 +17,6 @@
 
 """Python ecosystem specific routines."""
 
-from .digests_fetcher import DigestsFetcherBase
-from .digests_fetcher import PythonDigestsFetcher
-from .packages import Packages
-from .package_version import PackageVersion
-from .pipfile import Pipfile
-from .pipfile import PipfileMeta
-from .pipfile import PipfileLock
-from .project import Project
-from .source import Source
-from .aiosource import AIOSource, AsyncIterablePackages, AsyncIterableVersions, AsyncIterableArtifacts
```

Comment:
please dont remove them, add ether F401 or use `__all__` to let flake8 know about this.
if these imports are removed as a user of python package. 
i would have to use `thoth-python.digests_fetcher import DigestsFetcherBase` while i could use just `from thoth-python import DigestsFetcherBase`

some reference: https://stackoverflow.com/questions/31079047/python-pep8-class-in-init-imported-but-not-used

Reply:
thanks, they're back